hydromt_wflow/wflow.py

```diff
@@ -2051,7 +2053,8 @@ def setup_lakes(
 
         # add to grid
         rmdict = {k: self._MAPS.get(k, k) for k in ds_lakes.data_vars}
-        self.set_grid(ds_lakes.rename(rmdict))
+        ds_lakes = ds_lakes.rename(rmdict)
+        self.set_grid(ds_lakes)
```

Comment:
The original names should be kept here (ie revert your change) for when we update the names in the toml file a couple of lines later